airflow/providers/amazon/aws/operators/batch.py

```diff
@@ -133,17 +143,42 @@ def __init__(
         self.job_name = job_name
         self.job_definition = job_definition
         self.job_queue = job_queue
-        self.overrides = overrides or {}
-        self.array_properties = array_properties or {}
+
+        self.container_overrides = None
+        if overrides:
+            self.container_overrides = overrides
+            warnings.warn(
+                "Parameter `overrides` is deprecated, Please use `container_overrides` instead.",
+                DeprecationWarning,
+                stacklevel=2,
+            )
+            if container_overrides:
+                raise AirflowException(
+                    "If providing `container_overrides`, then old parameter 'overrides' should be removed."
+                )
```

Comment:
Can you add a comment explaining what's happening here?

Reply:
it's a regular deprecation thing, if users migrate to the new parameter they should remove the old one. I can try to rephrase to make it clearer